<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Viewer</string>
  </property>
  <property name="statusTip">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="openBtn">
          <property name="text">
           <string>open</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveBtn">
          <property name="text">
           <string>save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="zoomOutBtn">
          <property name="text">
           <string>zoom out</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cropBtn">
          <property name="text">
           <string>crop</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="resetBtn">
          <property name="text">
           <string>reset</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>30</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>rotate</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="angleHSlider">
          <property name="minimum">
           <number>-180</number>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="angleSpinBox">
          <property name="minimum">
           <number>-180</number>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
